tests/end_to_end/netsh_test.cpp

```diff
@@ -464,18 +493,69 @@ TEST_CASE("show maps", "[netsh][maps]")
     Platform::_close(outer_map_fd);
 }
 
+TEST_CASE("show links", "[netsh][links]")
+{
+    _test_helper_libbpf test_helper;
+
+    // Load and attach a program.
+    int result;
+    std::string output = _run_netsh_command(handle_ebpf_add_program, L"tail_call.o", L"pinned=none", nullptr, &result);
+    REQUIRE(strcmp(output.c_str(), "Loaded with ID 196609\n") == 0);
+    REQUIRE(result == NO_ERROR);
+
+    output = _run_netsh_command(handle_ebpf_show_links, nullptr, nullptr, nullptr, &result);
+    REQUIRE(result == NO_ERROR);
+    REQUIRE(
+        output == "\n"
+                  "   Link  Program\n"
+                  "     ID       ID\n"
+                  "=======  =======\n"
+                  " 327681   196609\n");
+
+    output = _run_netsh_command(handle_ebpf_delete_program, L"196609", nullptr, nullptr, &result);
+    REQUIRE(output == "");
+    REQUIRE(result == NO_ERROR);
```

Comment:
suggestion: should we add another call to `handle_ebpf_show_links` and check it shows no links now.

Reply:
Done